### PR TITLE
Custom user agent for api requests

### PR DESCRIPTION
**Pull request by simonlsk**

### CODE REVIEW

File: dagshub/streaming/filesystem.py [hunk 1]
```diff
@@ -481,10 +481,11 @@ def scandir(self, path='.'):
     @cache_by_path
     def _api_listdir(self, path: str, include_size: bool = False):
         return requests.get(f'{self.content_api_url}/{path}', auth=self.auth,
-                            params={'include_size': 'true'} if include_size else {})
+                            params={'include_size': 'true'} if include_size else {},
+                            headers=config.requests_headers)
 
     def _api_download_file_git(self, path: str):
-        return requests.get(f'{self.raw_api_url}/{path}', auth=self.auth)
+        return requests.get(f'{self.raw_api_url}/{path}', auth=self.auth, headers=config.requests_headers)
```
Comment (guysmoilov) [inline dagshub/streaming/filesystem.py:488]: You're sure that setting `headers=` won't cause other headers that SHOULD have been sent by default, to be omitted?

**Review by guysmoilov — APPROVED** [review]

File: dagshub/streaming/filesystem.py (hunk 1 above)
Comment (simonlsk) [reply dagshub/streaming/filesystem.py:488]: Yes, I checked that, they have a merge function of default + user requested.

File: setup.py [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import setuptools
+import dagshub.version
 
 with open("README.md", "r", encoding="utf8") as fh:
     long_description = fh.read()
 
 setuptools.setup(
     name="dagshub",
-    version="0.1.99-alpha",
+    version=dagshub.version.__version__,
```
[the remark this answers is not in the document]
Comment (simonlsk) [reply setup.py:9]: Yes

**Comment by guysmoilov** [comment]
OK, merging

**Pull request merged**